src/Symfony/Component/Cache/Tests/Traits/RedisProxiesTest.php

```diff
@@ -34,11 +34,11 @@ public function testRedisProxy($class)
         $methods = [];
 
         foreach ((new \ReflectionClass(\sprintf('Symfony\Component\Cache\Traits\\%s%dProxy', $class, $version)))->getMethods() as $method) {
-            if ('reset' === $method->name || method_exists(LazyProxyTrait::class, $method->name)) {
+            if ('reset' === $method->name || method_exists(LazyProxyTrait::class, $method->name) || $method->isInternal()) {
```

Comment:
This change is required to make the tests fail when our proxies miss a method. The other changes in this test reduce the diff with the version in the `7.4` branch.